src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_stats_test.py

```diff
@@ -99,6 +99,7 @@ def test_parse_log(self):
         'strategy_corpus_mutations_ml_rnn': 0,
         'strategy_corpus_subset': 50,
         'strategy_fork': 1,
+        'strategy_radamsa_mutator_plugin': 0,
```

Comment:
nit: maybe `strategy_mutator_plugin_radamsa`? to be similar to `strategy_corpus_mutations_ml_rnn` and `strategy_corpus_mutations_radamsa`

Reply:
Done